### DISCUSSION TITLE
Colorblind themes

### DISCUSSION THREAD

**Discussion by osxd** [community/community#48700] [Accessibility]
### Select Topic Area

Product Feedback

### Body

Hi!

It's really cool to see you working on "Colorblind themes" feature. Yet, this is not covering now the Code Diffs – and still using subtle red-green colors, which still easy to mix. Some contrast is needed (perhaps crossed background for deleted lines, font weight or stronger indication of "-" minus sign in the front of deleted lines.

UPD: in dark mode it might be even harder with blue-yellowish changes highlight.

**Comment by cehfisher** [comment]
Hi @osxd - thank you for providing this feedback! We wanted to follow up on this issue. Have the recent theme changes resolved this issue for you? If yes, could you please close out this issue? If not, what additional changes are needed? Please supply detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
